app/components/Page404/Page404.jsx

```diff
@@ -4,9 +4,9 @@ import {connect} from "alt-react";
 import SettingsStore from "stores/SettingsStore";
 import Translate from "react-translate-component";
 
-const light = require("assets/logo-404-light.png");
-const dark = require("assets/logo-404-dark.png");
-const midnight = require("assets/logo-404-midnight.png");
+const light = require("assets/logo-404-light.png").default;
```

Comment:
It would be preferable to switch to plain ES imports altogether, non?

Reply:
In the new version, the resources imported by require will be under ".default" and cannot be accessed directly. I suggest you do some tests. Maybe there is a better way to deal with this. I only see this method at present.

Reply:
Maybe it can be like this:
const {default  as light} = require("assets/logo-404-light.png")